pilot/pkg/features/pilot.go

```diff
@@ -251,6 +251,26 @@ var (
 			"Istio Resources",
 	).Get()
 
+	EnableStatus = env.RegisterBoolVar(
+		"PILOT_ENABLE_STATUS",
+		false,
+		"If enabled, pilot will update the CRD Status field of all istio resources with reconciliation status.",
+	).Get()
+
+	StatusQPS = env.RegisterFloatVar(
```

Comment:
See my other comments - if we don't already have a flag to control the rate and burst for ALL k8s operations, we should add it, and apply it to ALL operations, since there is a risk we'll overload apiserver.

Reply:
I think this merits further conversation.  In order to keep up with the rate of change, this feature needs a few hundred qps to itself.  I wonder if anyone has developed something like a QoS system for a shared client?  If not, perhaps we could implement such a thing.